internal/configparse/parse.go

```diff
@@ -165,7 +174,7 @@ func (pconf *parsedConfig) add(field string) {
 // newParsedConfig parses an input raw config as a runner.ConfigGlobal and returns
 // a parsedConfig or the first non-nil error occurring in the process.
 func newParsedConfig(uconf unmarshaledConfig) (parsedConfig, error) { //nolint:gocognit // acceptable complexity for a parsing func
-	const numField = 12 // should match the number of config Fields (not critical)
+	const numField = 13 // should match the number of config Fields (not critical)
```

Comment:
Suggestion (maybe out of scope but it comes to mind):
```go 
const numField = reflect.TypeOf(unmarshaledConfig{}).NumField()
```

Once and for all.

Reply:
Mmh, the call to `reflect.TypeOf` is probably more costly than the time benefit of predeclaring the slice capacity.
Another once and for all solution could be to stop predeclaring the slice capacity 😅